### ISSUE TITLE
Why not make a name mandatory?

### ISSUE THREAD

**Issue by nao027** [tbird20d/python-text-games#26]
If I press Enter without entering anything when you are asked for a name, the process will proceed, so why not make a name mandatory?

--
What is your name?
My name is:

**Comment by tbird20d** [comment]
I agree. There should be a test to make sure the name entered by the user is not empty.
